graphics/src/gradient.rs

```diff
@@ -0,0 +1,103 @@
+//! For creating a Gradient.
+use iced_native::Color;
+use crate::gradient::linear::Linear;
```

Comment:
Nit: Make this `pub use` so we can `use gradient::Linear` instead of `gradient::linear::Linear`